core/src/main/resources/defaults/blueprints/cdp-data-engineering-721.bp

```diff
@@ -204,6 +204,18 @@
           {
             "name": "tez.task.launch.cmd-opts",
             "value": "-XX:+PrintGCDetails -verbose:gc -XX:+UseNUMA -XX:+UseG1GC -XX:+ResizeTLAB"
+          },
+          {
+            "name": "tez.grouping.split-waves",
```

Comment:
Isn't it singular? tez.grouping.split-wave

Reply:
plural is correct.
https://github.infra.cloudera.com/CDH/tez/blob/cdpd-master/tez-mapreduce/src/main/java/org/apache/tez/mapreduce/grouper/TezSplitGrouper.java#L64

Have edited the jira description to fix the property name there.

Reply:
Cool. Thanks. Resolving.